src/layers/core/geojson-layer/geojson-layer.js

```diff
@@ -72,27 +72,32 @@ export default class GeoJsonLayer extends Layer {
     }
   }
 
-  _onHoverSublayer(info) {
```

Comment:
GeoJsonLayer64 still relies on _onHoverSublayer of its base layer so this change of name breaks it. I'll updated it for you.